### PR TITLE
Split CLI and module logger configuration

### PR DESCRIPTION
**Pull request by ablam**
- removed logger setup from __init__.py
- CLI is still using yaml config file + optional extra switch --log-filename to override file name
- module/package should handle only own logging routines, so there is dedicated module logger.py with two functions get_default_logger_config to obtain defaults (config is similar to the one from yaml but it doesn't touch root logger) and configure_logging (without params it will fallback to default config). It is possible to override filename and log format by modifying global variables DEFAULT_LOG_FORMAT and DEFAULT_LOG_FILE_PATH, otherwise logger's config dict needs to be obtained and modified.

example usage with defaults and changed file name: 

```
import suit_generator.logger

suit_generator.logger.DEFAULT_LOG_FILE_PATH = "/tmp/test.log"
suit_generator.logger.configure_logging()
```

example usage with custom params:
```
import suit_generator.logger

suit_generator.logger.DEFAULT_LOG_FILE_PATH = "/tmp/test.log"
logger_config = suit_generator.logger.get_default_logger_config()
logger_config['loggers'].update({
   'root': {
      'level': 'DEBUG',
      'handlers': ['file', 'console'],
    }
})
suit_generator.logger.configure_logging(logger_config)
```

### CODE REVIEW

File: suit_generator/cli.py [hunk 1]
```diff
@@ -28,9 +33,36 @@
 }
 
 
+def configure_cli_logging(log_file_name: str = None):
+    """
+    Configure logging for CLI.
+
+    :param log_file_name: log file name to be used (ovveride default log file name from logging.yaml)
```
Comment (jnsgsr) [inline suit_generator/cli.py:40]: typo in "ovveride", should be "override"

File: suit_generator/cli.py [hunk 2]
```diff
@@ -28,9 +33,36 @@
...
+    """
+    dir_path = pathlib.Path(os.path.dirname(os.path.realpath(__file__)))
```
Comment (jnsgsr) [inline suit_generator/cli.py:42]: I suggest to not mix os.path and pathlib.Path in the same module, generally in recent versions of python standard libraries pathlib.Path should contain all the needed functionality that was before in os.path. In this case I think it should be `dir_path = Path(__file__).resolve().parent`

**Review by jnsgsr — APPROVED** [review]

File: suit_generator/cli.py (hunk 1 above)
Comment (ablam) [reply suit_generator/cli.py:40]: fixed

File: suit_generator/cli.py (hunk 2 above)
Comment (ablam) [reply suit_generator/cli.py]: it was in __init__ file - I've just moved it - fixed

**Comment by ablam** [comment]
On second thought and after discussion, I've decided to remove module related config, because this should be done on TAS side according to needs. I also simplified CLI config file - there is no need to specify handler / verbosity per each sublogger, since log messages will propagate to the parent loggers.

**Review by robertstypa — APPROVED** [review]

**Review by jnsgsr — APPROVED** [review]

**Review by tomchy — APPROVED** [review]

**Pull request merged**